server/tso/allocator_manager.go

```diff
@@ -309,7 +320,10 @@ func (am *AllocatorManager) allocatorLeaderLoop(ctx context.Context, allocator *
 				zap.String("dc-location", allocator.dcLocation),
 				zap.Int("suffix", suffix),
 				errs.ZapError(err))
-			time.Sleep(200 * time.Millisecond)
+			// PD leader hasn't been elected out, wait for the campaign
+			if !longSleep(ctx, time.Second) {
```

Comment:
Because the PD leader campaign usually lasts for seconds, and I notice this line print a lot of logs during the test, so I increase the sleep time here.